Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -46,6 +49,9 @@ FullDiagnostics::FullDiagnostics (int i, std::string name):
     m_solver_deposits_current{
         !(WarpX::electromagnetic_solver_id == ElectromagneticSolverAlgo::None &&
         WarpX::electrostatic_solver_id != ElectrostaticSolverAlgo::LabFrameElectroMagnetostatic)}
+    #if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+    , m_apply_filter_rho{true}
+    #endif
```

Comment:
If we remove `m_apply_filter_rho` in `FullDiagnostics.H` as I suggested, then it would also be removed here.
```suggestion
```